tests/treeTest.py

```diff
@@ -0,0 +1,18 @@
+from coalahtml.tree import Tree
+import unittest
+
+
+class coalaHTMLTest(unittest.TestCase):
+
+    def setUp(self):
+        self.FL = [
+                    "/home/tushar/coala-html/tests",
+                    "/home/tushar/coala-html/coalahtml",
+                    "/home/tushar/coala-html/setup.py",
+                    "/home/tushar/coala-html/tests/TestUtilities.py"
+                ]
```

Comment:
Will it work with multiple levels? Like several more in tests dir. I think it may not.